README.md

```diff
@@ -37,7 +37,7 @@ From source:
 ```bash
 git clone https://github.com/MrLeeh/pyads.git --recursive
```

Comment:
Can you fix the repo URL (not correct)
https://github.com/stlehmann/pyads

Reply:
done thanks